samples/msgext-bot-SPUploader/SPUploader/appsettings.json

```diff
@@ -9,13 +9,14 @@
   "SPTenantId": "<<SHAREPOINT-TENANT-ID>>",
   "SharePointClientID": "<<SHAREPOINT-CLIENT-ID>>",
   "SharePointClientSecret": "<<SHAREPOINT-CLIENT-SECRET>>",
-  "SharePointEndPoint": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM>>",
-  "SharePointResource": "<<YOUR-SHAREPOINT-RESOURCE>>",
-  "SharePointBaseURL": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM/YOUR-APP-NAME>>",
+  "SharePointEndPoint": "<<YOUR-SHAREPOINT-DOMAIN.SHAREPOINT.COM eg: xxxxxxxxx.SHAREPOINT.COM>>",
+  "SharePointResource": "00000003-0000-0ff1-ce00-000000000000",
```

Comment:
is it a constant for any tenant or any organisation?

Reply:
yes, it is constant for all.
https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/authorization-code-oauth-flow-for-sharepoint-add-ins